apps/desktop/src/components/settings/views/sound.tsx

```
@@ -69,46 +70,165 @@ export default function Sound() {
+        throw error;
+      }
+    },
+    enabled: micPermissionStatus.data === true,
+  });
+
+  const microphoneDevices = useQuery({
+    queryKey: ["microphoneDevices"],
+    queryFn: async () => {
+      const result = deviceQuery.data;
+      console.log("Processing device query result:", result);
+      if (result && result.startsWith("DEVICES:")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)